Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -391,6 +391,7 @@
       <DependentUpon>AddEnvironmentDialog.xaml</DependentUpon>
```

Comment:
I've put some review comments in CondaEnvCreateInfoBar.cs, but they apply to the other 2 classes as well. Such as "The "Project?.Caption" contains a property "buildProject" which can be null..." and "Declare the "projectOrWorkspacename..."

Reply:
In the future, it would also be nice if you could separate the component changes into different components. One commit for package install, another for virtual env creation, etc. It make it easier to code review :smile: